Source/SpacePort/SpacePort/DataContext.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SpacePort.Models
+{
+    class DataContext: DbContext
+    {
+        public DbSet<Ticket> Ticket { get; set; }
+        public DbSet<Receipt> Receipt { get; set; }
+        protected override void OnConfiguring(DbContextOptionsBuilder optionBuilder)
+        {
+            optionBuilder.UseSqlServer(@"SERVER = den1.mssql7.gear.host; DATABASE = spaceportdb; UID = spaceportdb; PWD = Gk03R!351K-W");
```

Comment:
You probably already know this but you can use configuration files to hide sensitive data (connection strings for example) in open source code.

Reply:
Ye, our maingoal was just to fix the database and get that up working, next step is configuration files :)